apps/src/templates/instructions/codeReview/ReviewNavigator.jsx

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import DropdownButton from '@cdo/apps/templates/DropdownButton';
+import Button from '@cdo/apps/templates/Button';
+import javalabMsg from '@cdo/javalab/locale';
+
+class ReviewNavigator extends Component {
+  static propTypes = {
+    peers: PropTypes.array,
+    onSelectPeer: PropTypes.func,
+    onReturnToProject: PropTypes.func,
+    viewPeerList: PropTypes.bool,
+    loadError: PropTypes.bool
+  };
+
+  getPeerList() {
+    const {loadError, peers, onSelectPeer} = this.props;
+    if (loadError || !Array.isArray(peers)) {
```

Comment:
Won't react also throw an error if `peers` is not an array?

Reply:
Tested this out to make sure:
* I was originally intending to check whether `peers` is undefined, which is a valid value for this prop. When it is undefined, this check returns false and we see the expected error state.
* When I passed in another type for peers (i.e. an object) react raised a warning but still displayed the component. This check returned false and we again see the expected error state.
* I then tried passing in an object for `peers` and disabling this check. The code instead hit an error when trying to access peers.map. This led to 7 errors in the console, one of which tried to unmount the ProtectedStatefulDiv. This led to a rather fatal error where the entire window went blank.